drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodListDialogFragment.kt

```diff
@@ -162,4 +183,12 @@ class PaymentMethodListDialogFragment : DropInBottomSheetDialogFragment(), Payme
         val paymentComponentState = GenericComponentState(paymentComponentData, true, true)
         protocol.requestPaymentsCall(paymentComponentState)
     }
+
+    private fun collapseNotUsedUnderlayButtons(recyclerView: RecyclerView, draggedItem: AdyenSwipeToRevealLayout) {
```

Comment:
Can you explain why this is needed? I think I'm missing something.

Reply:
Whenever shopper swipes an item, we collapse the rest of the items automatically to have only one delete button visible at a given time. Also you can collapse them by clicking in the main area, but this is used if they don't collapse it by themselves before swiping another.

Reply:
Ahh got it, thanks for the clarification!